rust/src/lib/nm/show.rs

```diff
@@ -151,9 +155,18 @@ pub(crate) async fn nm_retrieve(
                 } else {
                     None
                 };
-                if let Some(iface) =
+                if let Some(mut iface) =
                     iface_get(nm_dev, nm_conn, nm_saved_conn, lldp_neighbors)
                 {
+                    // Clear forwarding when unsupported by NetworkManager to prevent failures
+                    // if the user tries to apply the returned state as-is.
```

Comment:
Please add a `TODO` comment requesting to remove this code when the minimum NM supported version is >= 1.54

Reply:
I added the TODO comment.